reference-architectures/standard/deploy-arch-ibm-pvs-inf-standard.md

```diff
@@ -0,0 +1,133 @@
+---
+copyright:
+  years: 2023
+lastupdated: "2023-02-21"
+
+keywords:
+
+subcollection: deployable-reference-architectures
+
+authors:
+  - name: Arnold Beilmann
+
+version: v7.0.0
+
+deployment-url: url #TODO
+
+docs: https://cloud.ibm.com/docs/solution-guide #TODO
```

Comment:
I think this docs link is correct, although the solution guides are not in the public docs yet.